test/TestServerProjects/custom-baseUrl-paging/Generated/Operations/PagingRestClient.cs

```diff
@@ -245,7 +245,7 @@ internal HttpMessage CreateGetPagesPartialUrlOperationNextRequest(string account
             uri.AppendRaw(accountName, false);
             uri.AppendRaw(host, false);
             uri.AppendPath("/paging/customurl/", false);
-            uri.AppendPath(nextLink, false);
+            uri.AppendRaw(nextLink, false);
```

Comment:
I think we should keep this one as is.

Reply:
I take it back.

Reply:
Discussed offline. Needed to parsing values with both path and query in the same nextLink.